src/app/zap-templates/templates/app/helper.js

```diff
@@ -136,7 +136,7 @@ function asReadType(type)
 // List of all cluster with generated functions
 var endpointClusterWithInit = [
   'Basic', 'Identify', 'Groups', 'Scenes', 'Occupancy Sensing', 'On/Off', 'Level Control', 'Color Control', 'IAS Zone',
-  'Pump Configuration and Control'
+  'Pump Configuration and Control', 'Ethernet Network Diagnostics'
```

Comment:
I guess this change will auto generate the code to call emberAfEthernetNetworkDiagnosticsClusterServerInitCallback, right? anything other purpose?

Reply:
Yes, generating that call is the only reason this part is here.